lib/hls/hls_parser.js

```diff
@@ -1452,12 +1452,15 @@ shaka.hls.HlsParser = class {
     const hlsSegments = playlist.segments;
     /** @type {Array.<!shaka.media.SegmentReference>} */
     const references = [];
-    let exisitingRefCount = 0;
+    // number of segment references in both the last and the current playlist
```

Comment:
Nit: Since this is describing what a variable does, it would be better as a JSDoc @type tag.

For example:
```js
/**
 * The number of segment references in both the last and the current playlist.
 * @type {number}
 */
```

Reply:
Added.